modules/preferences/test/spec.coffee

```diff
@@ -54,7 +54,7 @@ describe 'hx-preferences', ->
 
     describe 'timezone', ->
       it 'setter/getter', ->
-        hx.preferences.timezone('UTC+01:00').should.equal(hx.preferences)
```

Comment:
This is precisely one of the things for which tests are broke under a different timezone from GTM+00:00.
There is need for additional debugging, which I will do in my free time.